include/ck/tensor_operation/gpu/thread/threadwise_tensor_slice_transfer_v3r3.hpp

```diff
@@ -343,8 +344,27 @@ struct ThreadwiseTensorSliceTransfer_v3r3
 
                 // do data transpose
                 // TODO type_convert is not used yet!!!!!
-                transpose_vectors<SrcData, DstScalarPerVector, SrcScalarPerVector>{}(
-                    src_vector_refs, dst_vector_refs);
+                transpose_convert_vectors<SrcData,
+                                          DstData,
+                                          DstScalarPerVector,
+                                          SrcScalarPerVector>{}(src_vector_refs, dst_vector_refs);
+            });
+        }
+        else if constexpr(SrcVectorDim == DstVectorDim && SrcScalarPerVector % 2 == 0 &&
```

Comment:
Not needed. This PR will deal with this scenario properly.

https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/206